book/src/drivers.md

```diff
@@ -12,6 +12,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Alicloud Disk](https://github.com/AliyunContainerService/csi-plugin) | `diskplugin.csi.alibabacloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alicloud Disk | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
 [Alicloud NAS](https://github.com/AliyunContainerService/csi-plugin) | `nasplugin.csi.alibabacloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alicloud Network Attached Storage (NAS) | Persistent | Read/Write Multiple Pods | No | 
 [Alicloud OSS](https://github.com/AliyunContainerService/csi-plugin)| `ossplugin.csi.alibabacloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alicloud Object Storage Service (OSS) | Persistent | Read/Write Multiple Pods | No | 
+[Alluxio](https://github.com/Alluxio/alluxio) | `csi.alluxio.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alluxio File System) | Persistent | Read/Write Multiple Pod | Yes | 
```

Comment:
https://github.com/Alluxio/alluxio is a repo for the storage system itself.  You can have a hyperlink for it in the description column.  Usually the name column should have a hyperlink to your CSI driver repo.

Reply:
The CSI driver is supported in the main repo. I updated the link to be the directory that contains the csi code.